tests/ui/resolve/custom-inner-attribute.rs

```diff
@@ -0,0 +1,9 @@
+//@ check-pass
```

Comment:
Could you add a comment saying that the test checks that macro names resolved from libstd prelude still work if there's a crate level custom inner attribute?
And rename the test to something less generic.